plasmapy/utils/decorators/validators.py

```diff
@@ -366,15 +366,6 @@ def _validate_quantity(self, arg, arg_name: str, arg_validations: Dict[str, Any]
             and not arg_validations["pass_equivalent_units"]
         ):
 
-            if not arg.unit.is_equivalent(unit, equivalencies=None):
-                # non-standard conversion
-                warnings.warn(
-                    ImplicitUnitConversionWarning(
-                        f"{err_msg} has a non-standard unit conversion..."
-                        f"converting {arg.unit} to {unit}"
-                    )
-                )
-
```

Comment:
After waking up and coming here to make a suggestion and see you already did what I was thinking...
https://youtu.be/96FlAMoc_pY?t=35

I do think getting rid of the `implicitUnitConversionWarning` altogether is the way to go.  If someone specifies an equivalency, then they're expecting that functionality so there's no real reason to warn about it.

I think the documentation page could probably do better at explaining the unit conversion without so much jargon.

Reply:
Awesome! So I just go through the leftovers as mentioned in the main post and this is good to go, right?